.mergify.yml

```diff
@@ -130,3 +130,18 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
+
+  - name: backport patches to 9.1 branch
```

Comment:
Codereview: Why does the 8.19 section look different than the other?

Reply:
https://github.com/elastic/logstash/pull/17551 introduced the defaults, so those specific config entries per backport pull_request rule is not needed anymore.